drivers/cc2420/include/cc2420_params.h

```diff
@@ -30,54 +30,52 @@ extern "C" {
  * @{
  */
 #ifndef CC2420_PARAM_SPI
-#define CC2420_PARAM_SPI        (SPI_DEV(0))
+#define CC2420_PARAM_SPI        SPI_DEV(0)
```

Comment:
Here and below, you removed the parenthesis, thus I don't know what this is about, if keeping them, removing them... I don't know it there's a consensus on its use, but the changes in the PR suggest that you have a shared preference on them. I don't mind if they're there or not, it's just a bit strange to see this kind of changes.

Reply:
So, just to clarify, on which kind of defines do you prefer parenthesis ?